packages/core/tests/integration/components/field-renderer-test.js

```diff
@@ -147,17 +150,76 @@ module('Integration | Component | field-renderer', function(hooks) {
       @setNeededWhenEmbedded={{action noop}}
       @setPosition={{action noop}}
       @removeField={{action noop}}
+      @isRightEdgeField={{true}} {{!-- Burcu please delete this line when you merge your stuff --}}
```

Comment:
@burieberry Please make sure to delete this line from the tests after you rebase your work to remove the `@isRightEdgeField`